cloudconfig/podcfg/podcfg.go

```diff
@@ -49,6 +58,10 @@ type ControllerPodConfig struct {
 	// ControllerTag identifies the controller.
 	ControllerTag names.ControllerTag
 
+	// MachineNonce is set at provisioning/bootstrap time and used to
+	// ensure the agent is running on the correct instance.
+	MachineNonce string
```

Comment:
PodNonce?
But really, I don't think long term we even want this for k8s
Add a TODO to remove?

Reply:
changed